main/src/ca/uwaterloo/flix/runtime/interpreter/Channel.java

```diff
@@ -48,11 +48,20 @@
    */
   private int bufferSize;
```

Comment:
can this be final?

Reply:
We change it in the constructor if we're given a size of 0 where we use size 1 and set the unbuffered flag, so i don't think so?

Reply:
We've made a new commit changing this to final